samples/arm-none-eabi/reported.by

```diff
@@ -0,0 +1,310 @@
+reporter_name="Jasmin Jessich"
+reporter_url="http://jasmin.anw.at/"
+reporter_comment="NOTE: Please read samples/arm-none-eabi/reported.by prior to any built!"
+
+#
+# This configuration will build an arm-none-eabi compiler to be used for
+# embedded development. The original build script can be found here:
+#    https://launchpad.net/gcc-arm-embedded
+# To see the original build script, you have to download the installer:
+#    https://launchpad.net/gcc-arm-embedded/5.0/5-2015-q4-major
+# 
+# I tried to mimic the original configure options and environment settings with
+# crosstool-ng (CT_NG) as close as possible. I really analysed every option
+# which was not provided by CT-NG or which was added. The description in this
+# file explains each diversion from the original.
+# 
+# NOTE:   newlib-nano is currently not built!
+# 
+# The original uses the git/SVN versions of binutils-gdb, gcc and newlib. CT-NG
+# doesn't currently support using a git or SVN repository for the source
```

Comment:
> HEAD of ct-ng does support Git (see, for example, mingw.sh or binutils.sh). You'll need to
> modify the corresponding build script for gcc, though.

For the gcc 4.9.3 version I used git, but for 5.3.0 all required things are already in the upstream packages. I decided to skip the armv8-m support (this would require the latest changes in the git repos) and to stick with the official packages for now.

> Also curious, why not update the existing arm-unknown-none-eabi? I think having both
> arm-unknown-none-eabi and arm-none-eabi would be confusing.

When you look to the CT-NG options, you will see the difference. My goal was to provide a configuration which is as close as possible to the ARM embedded gcc version (arm-none-eabi). Maybe it would be better to rename my sample to arm-none-eabi-embedded, so it is clear what it does.